service_container.rst

```diff
@@ -923,6 +923,34 @@ them will not cause the container to be rebuilt.
     means that all classes are "available to be *used* as services" without needing
     to be manually configured.
 
+The namespace Option
+--------------------
+
+.. versionadded:: 3.4
+    The ``namespace`` option was added in Symfony 3.4.
+
+When using the ``resource`` option in YAML, the namespace prefix can only be used once
+per file when defining it as the ``id``. In order to have multiple definitions in the
+same file with the same namespace prefix, you can use the ``namespace`` option. When
+this option is used, the ``id`` can be anything as long as it is unique. For example,
```

Comment:
IMO this sentence is confusing. One may ask: Where is the id now taken from?

Reply:
Should I remove the sentence?